comixed-webui/src/app/collections/pages/collection-list/collection-list.component.ts

```diff
@@ -106,12 +106,26 @@ export class CollectionListComponent
 
   onShowCollection(entry: CollectionListEntry): void {
     this.logger.debug('Collection entry selected:', entry);
-    this.router.navigate([
-      '/library',
-      'collections',
-      this.routableTypeName,
-      entry.name
-    ]);
+    if (this.collectionType === CollectionType.SERIES) {
+      const seriesName = entry.name.substr(0, entry.name.length - 6);
+      const volume = entry.name.substring(entry.name.length - 4);
+      this.logger.info('*** seriesName:', seriesName, ' volume:', volume);
+      this.router.navigate([
+        '/library',
+        'collections',
+        'series',
+        seriesName,
+        'volumes',
+        volume
```

Comment:
if we don't have a volume (vnull), the route(http://localhost:4200/library/collections/series/Superman/volumes/null) leads to nothing (no comics in the page).

Reply:
How about, in the case of a null volume, we ensure it only shows issues that have no defined volume? Replacing null with "----" gives us an explicit value to compare against when filtering.